integration/tools/validation/src/main/java/alluxio/cli/fuse/FuseCorrectnessValidation.java

```diff
@@ -0,0 +1,39 @@
+package alluxio.cli.fuse;
+
+import com.google.common.base.Preconditions;
+import org.apache.commons.cli.ParseException;
+
+/**
+ * This class is the entry class for fuse correctness validation tool.
+ */
+public class FuseCorrectnessValidation {
```

Comment:
Since the test is put under the tools/validation, can we follow their common practice?
if the common practice does not make sense, then change the defined practice?
e.g. Implement AbstractValidationTask
but their defined formats are also bad = = can take the useful part but left the others